subprojects/docs/src/samples/userguide/files/inputFiles/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+// tag::set-input-files[]
+task<JavaCompile>("compile") {
```

Comment:
:x: I would use the same layout in the Groovy script, that is a single enclosing `compile {}` block

All those repeated blocks don't add value to the Groovy sample and they would also clutter the Kotlin sample if we added them.